namui-cli/www/runWasm.js

```diff
@@ -1,5 +1,5 @@
 async function run() {
-    await wasm_bindgen("/bundle_bg.wasm");
+    await wasm_bindgen("bundle_bg.wasm");
```

Comment:
On electron, If we fetch `/some_file`, then electron will request `file:///C/some_file` instead `file:///C/path/to/electron_app/some_file`.
So fix it to relative path